src/Meshes/buffer.ts

```diff
@@ -1,19 +1,32 @@
-import { Nullable, DataArray } from "../types";
+import { Nullable, DataArray, IndicesArray } from "../types";
 import { ThinEngine } from "../Engines/thinEngine";
 import { DataBuffer } from './dataBuffer';
+import { Observer } from '../Misc/observable';
+
+// TODO: Try to retain old Buffer constructor
+// QUESTION: We have instanced and divisor on both VertexBuffer and Buffer... I guess different VertexBuffers could have different divisors as long as they are not views into the same buffer and used at the same time... but what do we want?

```

Comment:
yes this is what we want as we want to cover two kinds of vertex buffers:
- a vertex buffer can simply be a full buffer
- but it can also be just part of a buffer (interleaved)

Reply:
Did it answer your question?

Reply:
Yes. Thanks for reminding me! Divisor is per attribute and not per buffer, makes sense! I had forgotten about that.